MiguelAValdesG/Hackerrank/JavaHashTablesRansomNote.java

```diff
@@ -0,0 +1,74 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+class Result {
+
+  public static boolean checkMagazine(List<String> magazine, List<String> note) {
+    HashMap<String, Integer> wordsInMagazine = new HashMap<>();
+    HashMap<String, Integer> wordsInNote = new HashMap<>();
+    final Integer INIT_VALUE = 0;
+    final Integer ONE = 1;
+    final boolean CAN_BE_FORMED = true;
+    final boolean CAN_NOT_BE_FORMED = false;
+
+    for (String word : magazine) {
+      if (!wordsInMagazine.containsKey(word)) {
+        wordsInMagazine.put(word, INIT_VALUE);
+      }
+      Integer wordConcurrences = wordsInMagazine.get(word);
```

Comment:
wordConcurrences??? you mean occurrences right???

The more meaningful your code, the more readable